sklearn/ensemble/voting_classifier.py

```diff
@@ -348,4 +367,16 @@ def get_params(self, deep=True):
 
     def _predict(self, X):
         """Collect results from clf.predict calls. """
-        return np.asarray([clf.predict(X) for clf in self.estimators_]).T
+
+        all_predictions = []
+
+        for name, est in zip(self.named_estimators_, self.estimators_):
+            try:
+                predictions = est.predict(X)
+            except NotFittedError as e:
+                raise type(e)(
+                    'Error encountered in estimator %s. '
```

Comment:
Are you assuming that as long as we're still bound to Python 2 and can't use `raise from` that we should limit this helper to the NotFittedError case?